docs/cookbook/create-basic-grid.md

```diff
@@ -0,0 +1,429 @@
+# Create Basic Grid
+In this cookbook, we will create a new grid to display salesmen data in the administration.
+We will learn how to properly use grid factory, how to configure data displayed, and override the default data presentation.
+In the end, we will be able to even sort our grid by column, paginate large result sets and delete records.
+
+## Prerequisites
+* Provided we are going to list the salesmen, you need to create the Salesman entity and add some entries to the database table following [the cookbook](/docs/cookbook/adding-a-new-entity.md), first.
```

Comment:
> was this result of grammarly ?
Provided we are going to === In the case we want to ??? BUT OK

No, this is my rich English :smile: I thought it is a synonym for "Because", however, it seems that it is not...so fixed

> or prerequisites is section where there is you pronoun ?

tbh, I was not paying attention to this kind of styling so it might be inconsistent...